11:04:51.030 INFO Test execution started: third_test
11:04:51.030 INFO Browser: firefox-remote
11:04:51.031 INFO Environment: test_env
11:04:51.031 INFO Using data:
    url: http://{}:{}@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer=
11:04:53.302 INFO Navigate to: 'http://User1:Automation_1_3@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer='
11:04:53.612 INFO Waiting for 10 seconds
11:05:03.770 ERROR UnexpectedAlertPresentException: Alert Text: None
Message: Dismissed user prompt dialog: http://10.0.1.73:8001 is requesting your username and password.

Traceback (most recent call last):
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\test_runner\test_runner.py", line 173, in run_test
    self.test_module.test(execution.data)
  File "C:\Users\alexandr.may\testdir\projects\some_project\tests\third_test.py", line 18, in test
    for i in home_page.tabsValues():
  File "C:\Users\alexandr.may\testdir\projects\some_project\pages\home_page.py", line 18, in tabsValues
    return golemDriver.find_all(css='.navigation > nav > ul > li > a > span')
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\webdriver\extended_driver.py", line 203, in find_all
    return common._find_all(self, **kwargs)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\golem\webdriver\common.py", line 174, in _find_all
    webelements = self.find_elements_by_css_selector(css)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 613, in find_elements_by_css_selector
    return self.find_elements(by=By.CSS_SELECTOR, value=css_selector)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 995, in find_elements
    'value': value})['value'] or []
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 320, in execute
    self.error_handler.check_response(response)
  File "c:\users\alexandr.may\appdata\local\programs\python\python37-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 241, in check_response
    raise exception_class(message, screen, stacktrace, alert_text)
selenium.common.exceptions.UnexpectedAlertPresentException: Alert Text: None
Message: Dismissed user prompt dialog: http://10.0.1.73:8001 is requesting your username and password.


11:05:04.440 INFO Test Result: CODE ERROR
